stdlib/public/Concurrency/Actor.cpp

```diff
@@ -134,6 +134,11 @@ SWIFT_RUNTIME_DECLARE_THREAD_LOCAL(
 
 } // end anonymous namespace
 
+AsyncTask*
+swift::swift_task_get_active() {
+  return ActiveTask::get();
+}
```

Comment:
if we instead change getCurrentAsyncTask to do just that we can remove this one.